src/main/java/hudson/plugins/sonar/action/SonarCacheAction.java

```diff
@@ -37,16 +38,16 @@
   private List<ProjectInformation> lastProjInfo;
 
   public SonarCacheAction() {
-    this.infoByTaskId = new HashMap<>();
-    this.infoByUrl = new HashMap<>();
+    this.infoByTaskId = new HashMap<String, ProjectInformation>();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2293)